### PR TITLE
Feat[WEB-3]: add header components and content

### PR DESCRIPTION
**Pull request by GarnaA**

### CODE REVIEW

File: src/components/Header/components/Navbar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import styles from './navbar.module.scss'
+
+function Navbar() {
+  return (
+    <div className={styles.linksContainer}>
```
Comment (AndrZakhara) [inline src/components/Header/components/Navbar/index.jsx]: u can use classname container

File: src/components/Header/components/Logo/index.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { Link } from 'react-router-dom'
+import logo from '/src/assets/header/logo.svg'
+import subText from '/src/assets/header/subText.svg'
+import styles from './logo.module.scss'
+
+function Logo() {
+  return (
+    <div>
+      <Link className={styles.logo} to="/">
+        <img src={logo} alt="Logo" />
```
Comment (AndrZakhara) [inline src/components/Header/components/Logo/index.jsx]: use span/div like wrapper for images or svg's
image

File: src/components/Header/components/Language/language.module.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
+.icon {
```
Comment (AndrZakhara) [inline src/components/Header/components/Language/language.module.scss:1]: icon is not vertically centered

**Review by MyHKo — APPROVED** [review]

**Review by AndrZakhara — APPROVED** [review]

**Review by sofiiavoitkova — APPROVED** [review]

File: src/App.jsx [hunk 4]
```diff
@@ -1,14 +1,19 @@
+import { Routes, Route } from 'react-router-dom'
 import Default from '@components/Default'
 import Header from '@components/Header'
 import Footer from '@components/Footer'
 import styles from './app.module.scss'
```
Comment (AndrZakhara) [inline src/App.jsx:5]: It is better to place styles at the end of the imports list

**Review by AndrZakhara — APPROVED** [review]

**Pull request merged**